StringCatalogEnum/Sources/StringCatalogEnum/model.swift

```diff
@@ -6,4 +6,21 @@ struct XCStrings: Decodable {
     let version: String
     let strings: [String: String]
 }
-*/
\ No newline at end of file
+*/
+struct XCStrings: Decodable {
```

Comment:
In https://github.com/superarts/XStringCatalog/issues/1, we listed several possible formats. Do we need to document the `struct`s here with sample JSON?